src/main/resources/nubesgen/terraform/modules/app-service/main.tf.mustache

```diff
@@ -116,6 +116,46 @@ resource "azurerm_app_service" "application" {
 {{/addonApplicationInsights}}
 
     # These are app specific environment variables
+{{#runtimeQuarkus}}
+    "QUARKUS_HTTP_PORT"           = 80
+    "QUARKUS_PROFILE"             = "prod"
+  {{#databaseTypeSqlServer}}
+
+    "QUARKUS_DATASOURCE_JDBC_URL" = "jdbc:sqlserver://${var.database_url}"
+    "QUARKUS_DATASOURCE_USERNAME" = var.database_username
+    "QUARKUS_DATASOURCE_PASSWORD" = var.database_password
+  {{/databaseTypeSqlServer}}
+  {{#databaseTypeMysql}}
+
+    "QUARKUS_DATASOURCE_JDBC_URL" = "jdbc:mysql://${var.database_url}?useUnicode=true&characterEncoding=utf8&useSSL=true&useLegacyDatetimeCode=false&serverTimezone=UTC"
+    "QUARKUS_DATASOURCE_USERNAME" = var.database_username
+    "QUARKUS_DATASOURCE_PASSWORD" = var.database_password
+  {{/databaseTypeMysql}}
+  {{#databaseTypePostgresql}}
+
+    "QUARKUS_DATASOURCE_JDBC_URL" = "jdbc:postgresql://${var.database_url}"
+    "QUARKUS_DATASOURCE_USERNAME" = var.database_username
+    "QUARKUS_DATASOURCE_PASSWORD" = var.database_password
+  {{/databaseTypePostgresql}}
+  {{!#addonRedis}}
+  {{! TODO ADD REDIS SUPPORT FOR QUARKUS }}
```

Comment:
Why couldn't you do this? If there's no specific Redis support for Quarkus, let's use the same env variables as for non-Spring Java

Reply:
Yes, there is a specific support for Redis in Quarkus. But because I don't know Redis, I couldn't test it live (I've tested MySQL and Mongo in production, but not Redis).

The Quarkus variable is `QUARKUS_REDIS_HOSTS` and its format is `redis://[username:password@][host][:port][/database]`. So, following the existing code for Spring, it would be something like:

```
QUARKUS_REDIS_HOSTS = redis://var.azure_redis_password@var.azure_redis_host:6380
```

But I didn't test it, so I don't know if it really works.

Reply:
Can you try to do your best for this variable, and I'll test it. I'm guessing Quarkus fails if it can't connect to it, so it should be pretty obvious.